frontend/cypress/e2e/features/tests/estatistica.feature

```diff
@@ -1,12 +1,11 @@
-Feature: Estatísticas
+Feature: Statistics
     As an administrator
     I want to view various performance metrics
     So that I can monitor and analyze the system's usage and its profitability.
 
 
-Scenario: visualizar estatísticas do restaurante
-Given estou logado com nome "admin"
-And estou na página "Home"
-When acesso a página "Estatistica"
-Then estou na página "Estatistica"
-And consigo ver "totalUsers", "totalitems", "totalOrders", "monthOrders", "totalRevenue", "monthRevenue" e "averageTicket"
+Scenario: visualize statistics
+Given I am at page "Home"
+When I access page "Statistics"
+Then I am at page "Statistics"
+And I can see "totalUsers", "totalitems", "totalOrders", "monthOrders", "totalRevenue", "monthRevenue" e "averageTicket"
```

Comment:
What are "totalUsers", "totalItems"....

Reply:
They are the statistics data parametrized in attributes from the json object.